File: ornek1/firma.go
package main

import "fmt"

type Firma struct {
	firma    string
	adres    string
	Firma_Id int
}

func (f *Firma) FirmaEkle(adres string, firma string) {
	fmt.Println(adres, firma)
}
func (f *Firma) Bul(id int) Firma {
	var bulunanFirma Firma
	return bulunanFirma
}

File: ornek1/Firma.php
<?php

class Firma{
    public $firma;
    public $adres;
    private $firma_ID;
    public function __construct(){}
    public function __destruct(){}

    public function firmaEkle($adres,$firma){
        return $adres." ".$firma;
    }
    public function Bul($id){
        return $id;
    }
}